### DISCUSSION TITLE
Records - Navigation/Refresh Issue

### DISCUSSION THREAD

**Discussion by dawghouse73** [nmelhado/league-page#219] [Q&A]
I can't really test anything on our site as we have no league data to display for Records (dynasty start up in 2023), but I noticed in the Legends site that if I go:
League Info > Records
then "Season Records" (page shows - season 2022 records information))
then click "Playoffs" the page doesn't load - the "Playoffs" button highlights, but no data is loaded. 
![image](https://github.com/nmelhado/league-page/assets/133917715/4cf32876-4856-479d-88ab-780e697364a8)

I also tried this path:
League Info > Records
then staying on "Regular Season", I clicked "Season Records" (the 2022 Regular Season records are shown)..
then clicked on Playoffs and the page doesn't load  - the "Playoffs" button highlights, but no data is loaded. 
![image](https://github.com/nmelhado/league-page/assets/133917715/618398cc-d72f-4246-9163-b182f9351119)

Also, if I go League Info > Records .. then click Playoffs first...I see the All-Time Records information, but when I then click "Season Records", the page loads empty - the All-Time Records button stays highlighted, the "Season Records" button is dimmed, but no data is showing. 
![image](https://github.com/nmelhado/league-page/assets/133917715/4e4f7230-b77e-4455-84a8-b7e5d3e05ecc)

Just trying to get a handle on what I should see by referencing the Legends League. 
And should I see anything in my site yet even though we haven't had a season yet?
Is there a default placeholder like for the Standings page and it's "Preseason, No Standings Yet" message?
Or just a blank page?
And if it's a blank page, how can I get a default message for each of the sections:
1. Records/Regular Season/All-Time Records (available after week 1 of 2023)
2. Records/Regular Season/Season Records (available after the 2023 season is completed, I assume)
3. Records/Playoffs/All-time Records (available after round 1 of the 2023 playoffs)
4. Records/Playoffs/Season Records (available at the end of the 2023 Playoffs I assume)

Thanks in advance.
Shulaces Out Dan!

**Comment by nmelhado** [comment]
~~Oh boy, this is definitely a bug. Do you mind reporting it so that it shows up in the issues tab? When I get a free moment I try to tackle as many of these as I can. There are two other open ones so I'll see if I can get all 3 taken care of in the next few weeks.~~

Thanks for reporting @dawghouse73

**Comment by nmelhado** [comment]
Never mind, this has already been reported. Being tracked in https://github.com/nmelhado/league-page/issues/213. Follow that thread for updates!
